src/main/resources/fabric.mod.json

```
@@ -5,18 +5,34 @@
   "version": "${version}",
   "environment": "*",
   "license": "Apache-2.0",
-  "icon": "assets/legacy-fabric-api/icon.png",
+  "icon": "assets/legacy-fabric/icon.png",
   "contact": {
     "homepage": "https://legacy-fabric.github.io/",
     "issues": "https://github.com/Legacy-Fabric/fabric/issues",
     "sources": "https://github.com/Legacy-Fabric/fabric"
   },
   "authors": [
-    "Legacy Fabric",
-    "FabricMC"
+    {
+      "name": "Legacy Fabric/Quilt",
+      "contact": {
+        "homepage": "https://legacyfabric.net/",
+        "discord": "https://discord.com/invite/yBTpRXe",
```

```suggestion
        "discord": "https://legacyfabric.net/discord",
```
Link the permanent invite

---

Done.